### PR TITLE
add one parameter to the o2-hmpid-raw-to-digits-stream-workflow in order

### PR DESCRIPTION
**Pull request by fapfap69**
to exclude the production of statistic intermediate files

### CODE REVIEW

File: Detectors/HMPID/workflow/README.md [hunk 1]
```diff
@@ -56,7 +56,8 @@ Data processor options: HMP-DataDecoder:
   --root-file arg (=/tmp/hmpRawDecodeResults)
                   Name of the Root file with the decoding
                   results.
-  --fast-decode   Use the fast algorithm. (error 0.8%
+  --fast-decode   Use the fast algorithm. (error 0.8%)
+  --get-results   Switch on the output of statistic result files
```
Comment (davidrohr) [inline Detectors/HMPID/workflow/README.md]: Thx for the PR, wouldn't --get-results-statistics be a better name for the option?`Just --get-results seems a bit misleading to me.

Comment (fapfap69) [reply Detectors/HMPID/workflow/README.md]: done.
Now I need to redo the PR or not ?

Comment (davidrohr) [reply Detectors/HMPID/workflow/README.md]: What do you mean be redo? You have pushed to the branch of the PR, so the PR was automatically updated with your commit.

**Comment by fapfap69** [comment]
ok, thank you

File: Detectors/HMPID/workflow/src/DataDecoderSpec2.cxx [hunk 2]
```diff
@@ -341,7 +345,8 @@ o2::framework::DataProcessorSpec getDecodingSpec2(bool askDISTSTF)
     inputs,
     outputs,
     AlgorithmSpec{adaptFromTask<DataDecoderTask2>()},
-    Options{{"result-file", VariantType::String, "/tmp/hmpRawDecodeResults", {"Base name of the decoding results files."}},
+    Options{{"get-results-statistics ", VariantType::Bool, false, {"Generate intermediat output results."}},
```
Comment (davidrohr) [inline Detectors/HMPID/workflow/src/DataDecoderSpec2.cxx]: Here is a typo, there is a trailing space in the option name which should be removed.

**Review by davidrohr — APPROVED** [review]

**Comment by davidrohr** [comment]
last version failed only in the fullCI due to the space typo, which is fixed now. tested locally in FST. merging.

**Pull request merged**